connectors/src/api_server.ts

```diff
@@ -106,6 +107,11 @@ export function startServer(port: number) {
     "/connectors/:connector_id/resources/titles",
     getResourcesTitlesAPIHandler
   );
+  app.post(
+    // must be POST because of body
+    "/connectors/:connector_id/resources",
```

Comment:
as per IRL: nodes would be a better name since we use `resource` for our "models" now